### PR TITLE
FIx visible color & Tabbing

### PR DESCRIPTION
**Pull request by nnat425**

### CODE REVIEW

File: src/app/components/TabItem/TabItem.jsx [hunk 1]
```diff
@@ -113,7 +111,7 @@ class TabItem extends React.Component {
         <li key={`${j}`} value={tab.value} id={`tab${j}`} className={(this.state.selectedFacet === tab.value ? 'activeTab' : null) } role='presentation'>
          <a href={`#tab${j}`}
           id={`link${j}`}
-          tabIndex={!this.state.tabNumber ?  '0' : parseInt(this.state.tabNumber) === j ? null : -1}
+          tabIndex={this.state.selectedFacet === tab.value ? null : -1}
```
Comment (LucasCioffi) [inline src/app/components/TabItem/TabItem.jsx]: Is `tabIndex` doing anything for us?  I saw in one ticket that Willa doesn't want to see any positive tabIndexes, but I'm not sure if null or -1 do anything.

File: src/app/components/TabItem/TabItem.jsx [hunk 2]
```diff
@@ -37,7 +36,7 @@ class TabItem extends React.Component {
     this.setState({ tabNumber: newTabIndex.toString(), selectedFacet: selectedTab, tabValue: selectedTab, selectedFacetAnchor: tabAnchor});
     this.props.onClickApply(selectedTab);
     let newTab = this.links[newTabIndex];
-    window.location.replace(window.location.href.split('#')[0] + `#tab${newTabIndex}`);
+    window.location.replace('#tab' + newTabIndex.toString());
```
Comment (LucasCioffi) [inline src/app/components/TabItem/TabItem.jsx]: I would think that we'd want to keep the beginning part of the URL which is represented by this: `window.location.href.split('#')[0]`

Comment (nnat425) [reply src/app/components/TabItem/TabItem.jsx]: We are keeping the beginning of the URL. Line 39 possibly might not even be needed. There is some weird behavior currently. Nothing show stopping.

File: src/app/components/TabItem/TabItem.jsx (hunk 1 above)
Comment (nnat425) [reply src/app/components/TabItem/TabItem.jsx]: Yes we need it! https://www.w3schools.com/tags/att_global_tabindex.asp 

It essentially helps set the active tab.

**Pull request merged**